_posts/2021-07-16-SC21-workshop-cfp.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Call for Papers:  RSE-HPC-2021 Workshop at SC21
```

Comment:
This is going to choke because of the ":" in the title - try putting quotes around the whole thing.

Reply:
yeah I don't see any title here: https://1310-120906408-gh.circle-artifacts.com/0/usrse.github.io/2021-07-16-2021-07-16-SC21-workshop-cfp/index.html that could be it.

Reply:
Thanks!  I would have had no idea...

Reply:
Are these the SC colors (when I highlight text it's like a sea green)

![image](https://user-images.githubusercontent.com/814322/126016341-c57fc374-762e-4667-8023-c194b87de203.png)

Reply:
This is probably an issue for the other site, but if we are trying to match the colors here:
![image](https://user-images.githubusercontent.com/814322/126016375-62e54249-a665-41a4-835e-d98f864c56f1.png)

We should probably get the details right

Reply:
Yes, for the workshop site itself (which is in its own repo) I was aiming to match the colors for the SC21 site as closely as I could.

Reply:
Would you be open to a PR there? I'm thinking we could try a dark background (blue) and then have the same orange/green accents. I guess if others think it's okay it probably doesn't matter, but I'm thinking we could try to get a closer match.

Reply:
I guess glancing at them side by side it does look similar - maybe I should just let it be :)

Reply:
https://www.youtube.com/watch?v=QDYfEBY9NM4 :)

Reply:
Looks like the title is fixed after adding the quotes:  https://1311-120906408-gh.circle-artifacts.com/0/usrse.github.io/index.html

Reply:
If you dive down to one of the lower-level SC pages like https://sc21.supercomputing.org/presentation/?id=wksp140&sess=sess140, the match is easier to see.  I think the SC folks have changed the color of their headers since I first set up the workshop site a few months ago; if I recall correctly the SC headers were orange back then, and I copied that.

Reply:
yeah you are right!

![image](https://user-images.githubusercontent.com/814322/126017306-51a99a61-01bb-43e0-9161-0d083f46a167.png)